### ISSUE TITLE
Several `'item_name'`s required for CM portion of 'common_forms' tab in metadata.xlsx

### ISSUE THREAD

**Issue by aclark02-arcus** [openpharma/clinsight#53]
All of the below highlighted `item_name`s are required thanks to [this block](https://github.com/openpharma/clinsight/blob/f29a720bd6a705241adf76843a4fc5195a51c7c0/R/fct_tables.R#L264-L285) of code.

* ![image](https://github.com/user-attachments/assets/72df858c-395c-4564-8d07-dba298887bca)

**Comment by LDSamson** [comment]
@aclark02-arcus  I don't think it is highlighted. Maybe we should perform the highlighting in the actual metadata.xlsx file that is in the package right now, and mention them in the documentation.
